AGENTS.md

```
@@ -1,69 +1,274 @@
+
+### Step 2: Start Trigger Dev in Background
+
+```bash
+# in your triggerdotdev/references clone
+cd projects/hello-world && pnpm exec trigger dev
+# Wait for "Local worker ready [node]"
+```
+
+### Step 3: Trigger and Monitor Tasks via MCP
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)